motioneye/config.py

```diff
@@ -2142,6 +2149,7 @@ def _set_default_motion(data):
     data.setdefault('@admin_password', '')
     data.setdefault('@normal_username', 'user')
     data.setdefault('@normal_password', '')
+    data.setdefault('@lang', '')
```

Comment:
I suggest to use English as default language, understood by most users, and to not change the current behaviour:
```suggestion
    data.setdefault('@lang', 'en')
```